acceptance/clear-acceptance-space.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -eux
+
+cf api ${CF_API_URL}
+(set +x; cf auth $CF_USERNAME $CF_PASSWORD)
+
+cf target -o ${CF_ORGANIZATION}
+if ! cf space ${CF_SPACE}; then
+  exit 0
+fi
+
+cf target -s ${CF_SPACE}
+
```

Comment:
It looks like the CC will bubble those errors up and fail this delete: https://github.com/cloudfoundry/cloud_controller_ng/blob/f4e10737b752c36059bbd2e2858bf37b12925eb2/app/actions/space_delete.rb#L10-L27